packages/components/src/date-time/time/index.tsx

```diff
@@ -190,10 +190,11 @@ export function TimePicker( {
 
 	const monthField = (
 		<MonthSelectWrapper>
-			<MonthSelect
+			<SelectControl
```

Comment:
Replaced the styled `MonthSelect` with a plain `SelectControl` because we don't need the custom styling anymore.